sparse_strips/vello_hybrid/src/render/webgl.rs

```diff
@@ -600,20 +810,23 @@ impl WebGlPrograms {
                 WebGl2RenderingContext::TEXTURE_2D,
                 0,
                 WebGl2RenderingContext::RGBA8 as i32,
-                render_size.width as i32,
-                render_size.height as i32,
+                new_render_size.width as i32,
+                new_render_size.height as i32,
                 0,
                 WebGl2RenderingContext::RGBA,
                 WebGl2RenderingContext::UNSIGNED_BYTE,
                 None,
             )
             .unwrap();
 
-            self.render_size = render_size.clone();
+            self.render_size = new_render_size.clone();
         }
+    }
 
-        // Process alpha data for texture
+    /// Upload alpha data to the texture.
+    fn upload_alpha_texture(&mut self, gl: &WebGl2RenderingContext, alphas: &[u8]) {
```

Comment:
Similar comment to the WGPU comment: We don't necessarily need the `self.alpha_data` staging buffer + redundant copy (but happy to address this comment by adding a `TODO` to the code).